### ISSUE TITLE
Minefactory reloaded chunkloaders are dead

### ISSUE THREAD

**Issue by RubberSpark** [VoidFlame/Issues#33]
Why are chunk loaders not working anymore for me and other people to do a mass mining operation w/Digital miners we need to load the nearby chunks for it to work also chunk loaders h dont lag as far as i am aware

**Comment by kmecpp** [comment]
Chunk loaders will probably be banned or removed from the modpack all together. It's not a feature normal players should have.

**Comment by RubberSpark** [comment]
Could you make it so its a donor item only then like a high donor?

**Comment by kmecpp** [comment]
Depends on how other players feel about that

**Comment by donkeypoop** [comment]
yeah chunk loader would love to stay in the game so i think a donor should have access at the very least
